c10/util/BFloat16-inl.h

```diff
@@ -167,7 +167,7 @@ inline C10_HOST_DEVICE double operator-(double a, BFloat16 b) {
 inline C10_HOST_DEVICE double operator*(double a, BFloat16 b) {
   return a * static_cast<double>(b);
 }
-inline C10_HOST_DEVICE double operator/(double a, BFloat16 b) {
+inline C10_HOST_DEVICE double operator/(double a, BFloat16 b) __ubsan_ignore_float_divide_by_zero__ {
```

Comment:
Clang ASAN2 `test_variant_consistency_eager_linalg_norm_cpu_bfloat16` failed but is passing with g++.
Hopefully it'd pass with this change.